backend/src/main/java/com/woowacourse/levellog/levellog/application/LevellogService.java

```diff
@@ -26,32 +27,29 @@
 public class LevellogService {
 
     private final LevellogRepository levellogRepository;
+    private final LevellogQueryRepository levellogQueryRepository;
     private final TeamRepository teamRepository;
-    private final MemberRepository memberRepository;
     private final TimeStandard timeStandard;
 
     @Transactional
-    public Long save(final LevellogWriteDto request, final Long authorId, final Long teamId) {
+    public Long save(final LevellogWriteDto request, @Authentic LoginStatus loginStatus, final Long teamId) {
```

Comment:
```suggestion
    public Long save(final LevellogWriteDto request, @Authentic final LoginStatus loginStatus, final Long teamId) {
```
LoginStatus에 final이 빠진거 같아요! 👀